{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cve@mitre.org",
			"ID": "CVE-2022-47952"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "lxc-user-nic in lxc through 5.0.1 is installed setuid root, and may allow local users to infer whether any file exists, even within a protected directory tree, because \"Failed to open\" often indicates that a file does not exist, whereas \"does not refer to a network namespace path\" often indicates that a file exists. NOTE: this is different from CVE-2018-6556 because the CVE-2018-6556 fix design was based on the premise that \"we will report back to the user that the open() failed but the user has no way of knowing why it failed\"; however, in many realistic cases, there are no plausible reasons for failing except that the file does not exist."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "CWE-203"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1783591/comments/45",
					"tags": [
						"Issue Tracking",
						"Vendor Advisory"
					],
					"url": "https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1783591/comments/45"
				},
				{
					"name": "https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1783591/comments/45",
					"tags": [
						"Issue Tracking",
						"Vendor Advisory"
					],
					"url": "https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1783591/comments/45"
				},
				{
					"name": "https://github.com/lxc/lxc/blob/0b83d71c2c8f3bac9503f894cd84584f79258bb3/lxc.spec.in#L274",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://github.com/lxc/lxc/blob/0b83d71c2c8f3bac9503f894cd84584f79258bb3/lxc.spec.in#L274"
				},
				{
					"name": "https://github.com/lxc/lxc/blob/0b83d71c2c8f3bac9503f894cd84584f79258bb3/lxc.spec.in#L274",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://github.com/lxc/lxc/blob/0b83d71c2c8f3bac9503f894cd84584f79258bb3/lxc.spec.in#L274"
				},
				{
					"name": "https://github.com/lxc/lxc/blob/0b83d71c2c8f3bac9503f894cd84584f79258bb3/src/lxc/cmd/lxc_user_nic.c#L1085-L1104",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://github.com/lxc/lxc/blob/0b83d71c2c8f3bac9503f894cd84584f79258bb3/src/lxc/cmd/lxc_user_nic.c#L1085-L1104"
				},
				{
					"name": "https://github.com/lxc/lxc/blob/0b83d71c2c8f3bac9503f894cd84584f79258bb3/src/lxc/cmd/lxc_user_nic.c#L1085-L1104",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://github.com/lxc/lxc/blob/0b83d71c2c8f3bac9503f894cd84584f79258bb3/src/lxc/cmd/lxc_user_nic.c#L1085-L1104"
				},
				{
					"name": "https://github.com/MaherAzzouzi/CVE-2022-47952",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://github.com/MaherAzzouzi/CVE-2022-47952"
				},
				{
					"name": "https://github.com/MaherAzzouzi/CVE-2022-47952",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://github.com/MaherAzzouzi/CVE-2022-47952"
				},
				{
					"name": "[debian-lts-announce] 20230821 [SECURITY] [DLA 3533-1] lxc security update",
					"url": "https://lists.debian.org/debian-lts-announce/2023/08/msg00025.html"
				},
				{
					"name": "[debian-lts-announce] 20230821 [SECURITY] [DLA 3533-1] lxc security update",
					"url": "https://lists.debian.org/debian-lts-announce/2023/08/msg00025.html"
				}
			]
		}
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "LOCAL",
				"availabilityImpact": "NONE",
				"baseScore": 3.3,
				"baseSeverity": "LOW",
				"confidentialityImpact": "LOW",
				"integrityImpact": "NONE",
				"privilegesRequired": "LOW",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N",
				"version": "3.1"
			},
			"exploitabilityScore": 1.8,
			"impactScore": 1.4
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:linuxcontainers:lxc:*:*:*:*:*:*:*:*",
						"versionEndIncluding": "5.0.1",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-11-21T07:32Z",
	"publishedDate": "2023-01-01T06:15Z"
}
